anchore_engine/analyzers/syft/handlers/common.py

```diff
@@ -0,0 +1,44 @@
+from typing import DefaultDict, Union
```

Comment:
Nit: Do we have a consensus on whether these types of modules should be called `common`, `util`, or `utils`? We have all three in the codebase but I think they are all more or less the same thing.

(This is not at all a review-blocker comment, just interested in folks thoughts.)

Reply:
Yeah we definitely do not have a consensus on this. Maybe something we should bring up in the platform team meeting?